docs/getting-credentials-and-accounts.md

```diff
@@ -0,0 +1,55 @@
+> NOTE this document refers to Red Hat internal components
+
+# Quota Management with Account Management Service (AMS) SKU
+
+The [Account Management Service](https://api.openshift.com/?urls.primaryName=Accounts%20management%20service) manages users subscriptions. The fleet manager uses the service offered by AMS to manage quota. Quota comes in form of stock keeping unit (SKU) assigned to a given organisation. The process is the same as requesting SKU for addons which is described in the following link https://gitlab.cee.redhat.com/service/managed-tenants/-/blob/main/docs/tenants/requesting_skus.md.
+
+# User Account & Organization Setup
+
+1. Create new Red Hat organization and users belonging to this organization:
+* Make sure that you are logged out from any existing `access.redhat.com` sessions. Use the `register new organization` flow to create an organization
+* Create new users using user management UI. It is recommended to append or prefix usernames of your team members, e.g. `dffrench_control_plane` for `dffrench` username being a member of `Control Plane` team.
+2. An AMS Role defined through the OCM Resources mechanism has been defined, available for developers of the KAS Fleet Manager to get the needed permissions
+   to perform KAS Fleet Manager development. The definition of the role can be found in 
+   https://gitlab.cee.redhat.com/service/uhc-account-manager/-/blob/master/pkg/api/roles/managed_kafka_service.go.
+3. Users interested in developing KAS Fleet Manager should have this role assigned to their OCM account. This can be done using the OCM Resources
+   mechanism. An example of assigning the AMS role previously mentioned to a user can be found [here](https://gitlab.cee.redhat.com/service/ocm-resources/-/merge_requests/812).
+4. KAS Fleet Manager developers also need AMS quota for installing specific OCM add-ons that KAS Fleet Manager requires. An example showing the required quotas for the addons can be found [here](https://gitlab.cee.redhat.com/service/ocm-resources/-/blob/master/data/uhc-stage/orgs/13640203.yaml).
+
+# Vault
+For automation (fleet-manager deployments and app-interface pipelines) it is required to use [vault](https://gitlab.cee.redhat.com/service/app-interface#manage-secrets-via-app-interface-openshiftnamespace-1yml-using-vault) for secrets management
+
+# AWS
+AWS accounts are required for provisioning development OSD clusters and to access AWS Route53 for Domain Name System. To request a AWS accounts, have a chat with your team's manager.
+> NOTE, if you are in the Middleware, make sure to send an email to `mw-billing-leaders@redhat.com` request the account to be created. The email should follow the below format:
```

Comment:
are we ok to put this mailing list publicly?

Reply:
This change is not adding anything new, it was already public. Specifically, it was introduced in commit `d8a0e40c3e1082802261a62d8a251c79570946d0` in the README.md, and later reorganized on other docs.

Reply:
updated